src/sentry/api/serializers/rest_framework/sentry_app.py

```diff
@@ -100,16 +103,11 @@ def get_current_value(field_name):
         return get_current_value
 
     def validate_name(self, value):
-        if not value:
-            return value
-
-        queryset = SentryApp.with_deleted.filter(slug=slugify(value))
-
-        if self.instance:
-            queryset = queryset.exclude(id=self.instance.id)
-
-        if queryset.exists():
-            raise ValidationError(u"Name {} is already taken, please use another.".format(value))
+        # even though we technically don't have an issue renaming something to a really long time
+        # there's no reason we should allow this
+        max_length = 64 - UUID_CHARS_IN_SLUG - 1  # -1 comes from the - before the UUID bit
```

Comment:
Is it the same 64 as [here](https://github.com/getsentry/sentry/pull/16089/files#diff-3ca79d5457ed1f4d4e9586d642b26b0fR50)? Should we make it a constant?

Reply:
@manuzope Nah, those are only incidentally the same. The `64` in this case comes from the size of the column but the masked value is just something the client sees.